<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>常用工具</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>tools_settings_icon.ico</normaloff>tools_settings_icon.ico</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QPlainTextEdit" name="text_result"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="button_del">
            <property name="text">
             <string>del</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_add">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="button_7">
            <property name="text">
             <string>7</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_8">
            <property name="text">
             <string>8</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_9">
            <property name="text">
             <string>9</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_sub">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="button_4">
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_5">
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_6">
            <property name="text">
             <string>6</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_mul">
            <property name="text">
             <string>×</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="button_1">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_2">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_3">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_div">
            <property name="text">
             <string>÷</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="button_residual">
            <property name="text">
             <string>%</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_0">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_dot">
            <property name="text">
             <string>.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_eq">
            <property name="toolTip">
             <string>equals</string>
            </property>
            <property name="text">
             <string>=</string>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Backspace</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="actionNew_Ctrl_N"/>
    <addaction name="actionOpen_Ctrl_O"/>
    <addaction name="actionRecent"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Ctrl_s"/>
    <addaction name="actionSave_As_Ctrl_Shift_S"/>
    <addaction name="actionSave_All_Ctrl_Shift_S"/>
    <addaction name="separator"/>
    <addaction name="actionQuit_Ctrl_Q"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Ctrl_N">
   <property name="text">
    <string>New(Ctrl+N)</string>
   </property>
  </action>
  <action name="actionOpen_Ctrl_O">
   <property name="text">
    <string>Open(Ctrl+O)</string>
   </property>
  </action>
  <action name="actionRecent">
   <property name="text">
    <string>Recent</string>
   </property>
  </action>
  <action name="actionSave_Ctrl_s">
   <property name="text">
    <string>Save(Ctrl+s)</string>
   </property>
  </action>
  <action name="actionSave_As_Ctrl_Shift_S">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionSave_All_Ctrl_Shift_S">
   <property name="text">
    <string>Save All(Ctrl+Shift+S)</string>
   </property>
  </action>
  <action name="actionQuit_Ctrl_Q">
   <property name="text">
    <string>Quit(Ctrl+Q)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>button_1</tabstop>
  <tabstop>button_2</tabstop>
  <tabstop>button_3</tabstop>
  <tabstop>button_4</tabstop>
  <tabstop>button_5</tabstop>
  <tabstop>button_6</tabstop>
  <tabstop>button_7</tabstop>
  <tabstop>button_8</tabstop>
  <tabstop>button_9</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>button_add</tabstop>
  <tabstop>button_sub</tabstop>
  <tabstop>button_mul</tabstop>
  <tabstop>button_del</tabstop>
  <tabstop>button_div</tabstop>
  <tabstop>button_residual</tabstop>
  <tabstop>button_dot</tabstop>
  <tabstop>button_eq</tabstop>
  <tabstop>button_0</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
